spec/teaspoon/server_spec.rb

```diff
@@ -83,6 +92,12 @@
       subject.port = 31337
       expect(subject.url).to eq("http://127.0.0.1:31337")
     end
+
+    it "returns a url for the server that includes the host" do
+      subject.host = '0.0.0.0'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.